srcpkgs/dwarffortress/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'dwarffortress'
+pkgname=dwarffortress
+version=0.44.12
+revision=1
+archs="x86_64"
+create_wrksrc=yes
+depends="gtk+>=2 SDL>=1.2 SDL_ttf SDL_image libGL glu"
+short_desc="Control a dwarven outpost or an adventurer in a randomly generated, persistent world"
+maintainer="Robert Stancil <robert.stancil@mavs.uta.edu>"
+license="nonfree"
```

Comment:
license= should be "custom: Proprietary"

`custom:` prefix because it is not under SPDX